src/index.html

```diff
@@ -4,10 +4,581 @@
+                </div>
+
+                <div class="testimonials__name">
+                  Azadeh Hawkins
+                </div>
+
+                <div class="testimonials__role">
+                  Hawkins Consulting
+                </div>
+              </div>
+
+              <div class="testimonials__card">
+                <div class="testimonials__img">
+                  <img
+                    src="./images/testimonials/Photo (3).2819df65.png"
+                    alt="testimonial-img"
+                    class="testimonial__photo"
+                  >
+                </div>
+
+                <div class="testimonials__quotes">
+                  <img src="./images/testimonials/“.svg" alt="quotes">
+                </div>
+
+                <div class="testimonials__discription">
+                  DIA raises the agency bar to stratospheric heights on both creative output and client service.
+                </div>
+
+                <div class="testimonials__name">
+                  Michel Grover
+                </div>
+
+                <div class="testimonials__role">
+                  Hulu
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <div class="vision">
+        <div class="vision__container">
+          <div class="vision__title">
+            Vision, Passion, Results
+          </div>
+
+          <div class="vision__description">
+            We are sure that first-rate job is possible only if all three
+            <br class="br">
+            components are united.
+          </div>
+
+          <div class="vision__button button">
+            Apply
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="contact-us grid" id="contact-us">
+          <div class="message-block grid__column--1-5">
+            <div class="message-block__title">
+              Send us  a message
+            </div>
+
+            <form
+              action="#"
+              method="get"
+              class="message-block__form"
+            >
+              <input
+                type="email"
+                name="email"
+                placeholder="Your email"
+                required
+                class="message-block__form-email input"
+              >
+
+              <input
+                type="text"
+                name="name"
+                placeholder="Your name"
+                required
+                autocomplete="off"
+                class="message-block__form-name input"
+              >
+
+              <textarea
+                name="message"
+                placeholder="Your message"
+                required
+                autocomplete="off"
+                class="message-block__form-message input"
+              >
+              </textarea>
```

Comment:
I can't see a placeholder on a page:
![image](https://user-images.githubusercontent.com/37161315/141078924-079dd9d9-ae5b-4f88-a574-ceedd270f9d8.png)

It behaves so, cause you have one space between opening and closing tag, you can fix it by formatting it like that:
```suggestion
             ></textarea>
```

Reply:
Also textarea shouldn't be resizeable